Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/RobotCameraPipeline.java

```diff
@@ -44,6 +50,11 @@ public Mat processFrame(Mat input) {
         // mask the blurred frame
         Core.inRange(mat, lowerRange, upperRange, mat);
 
+        // invert ranges if looking for red
```

Comment:
The ranges have to be inverted for red because the red HSV range is on both ends of the spectrum, and so we are detecting from 20-160 hue and inverting it so we detect from 0-20 and 160-180

Reply:
That should be in a comment somewhere in the code since it's not very intuitive. Either here or in the constants where you define red. Probably in both.

Reply:
Added a comment detailing why the inversion is necessary.